pages/services/edge-lb/1.1/installing/index.md

```diff
@@ -17,48 +17,35 @@ To configure a service account and install the Edge-LB package, use the instruct
 - You have access to [the remote Edge-LB repositories](https://support.mesosphere.com/hc/en-us/articles/213198586).
 
 **Limitations**
-- Currently, Edge-LB works only with DC/OS Enterprise in permissive mode on DC/OS 1.10, and permissive or strict mode on DC/OS 1.11 [security mode](/latest/security/ent/#security-modes). 
+- Currently, Edge-LB does not work with DC/OS Enterprise in strict mode on DC/OS 1.10. All [security modes](/latest/security/ent/#security-modes) are supported with DC/OS 1.11 and later. DC/OS 1.9 or earlier is not supported.
```

Comment:
Can we make it in the order of release? Something like:
`Edge-LB supports all [security modes] in DC/OS 1.11 and later.  It supports Permissive, Disabled in DC/OS 1.10. DC/OS 1.9 or earlier is not supported.`